dev_requirements.txt

```diff
@@ -1 +1,2 @@
-Cython
+ipython
+Cython --install-option='--no-cython-compile'
```

Comment:
Travis don't like you pass the option like this. I don't know how to give it correctly.
I suggest that you let this file as it was before and add `pip install ipython` in `.travis.yml`.